### PR TITLE
SPEC-933: Clarify that hostnames in  URLs work like normal host specifications

### PR DESCRIPTION
**Pull request by derickr**
Also added better links to the test plan that was split out here, and that it builds on the Connection String spec.

### CODE REVIEW

File: source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst [hunk 1]
```diff
@@ -32,6 +32,12 @@ domain to return a list of host names. Supporting this feature would assist
 our users by decreasing maintenance load, primarily by removing the need to
 maintain seed lists at an application level.
 
+This specification builds on the `Connection String`_ specification. It adds a
+new protocol schema and modifies how the `Host Information`_ is interpreted.
```
Comment (mbroadst) [inline source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst]: `scheme` not `schema`

File: source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst [hunk 2]
```diff
@@ -32,6 +32,12 @@ domain to return a list of host names. Supporting this feature would assist
 our users by decreasing maintenance load, primarily by removing the need to
 maintain seed lists at an application level.
 
+This specification builds on the `Connection String`_ specification. It adds a
+new protocol scheme and modifies how the `Host Information`_ is interpreted.
+
+.. _`Connection String`: ../connection-string/connection-string-spec.rst
+.. _`Host Information`: ../connection-string/connection-string-spec.rst#host-information
```
Comment (ajdavis) [inline source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst]: Yeah, I guess we have to accept that we're never going to really publish these aside from uploading them to GitHub, so links within GitHub are fine.

Comment (mbroadst) [reply source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst]: Yeah I kind of thought these belonged in a "Terminology" section, but I think this is fine as well.

**Comment by derickr** [comment]
@mbroadst - Something odd about the schema/scheme typo fix that I made. In any case, I've fixed that so this PR can now be closed.

**Pull request closed without merging**